.github/workflows/tools-releases.yml

```diff
@@ -55,25 +41,29 @@ jobs:
               ETCD_VERSION=$(yq eval '.etcd' $changed_file)
               echo "ETCD_VERSION=$ETCD_VERSION" >> $GITHUB_ENV
             done
-      - name: Set up Docker
-        uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # tag=v3.3.0
-      - name: Log in to the Container registry
-        uses: docker/login-action@e92390c5fb421da1463c202d546fed0ec5c39f20 # tag=v3.1.0
+      - name: Build packages
+        run: |
+          make release-envtest \
+            KUBERNETES_VERSION=${{ env.KUBERNETES_VERSION }} \
+            GO_VERSION=${{ env.GO_VERSION }} \
+            ETCD_VERSION=${{ env.ETCD_VERSION }}
```

Comment:
Can we drop GO_VERSION and ETCD_VERISION? (and the code above setting them)

Reply:
Yes